cluster/node-pools/worker-splitaz/userdata.yaml

```diff
@@ -46,6 +46,9 @@ write_files:
                - get-token
                - --cluster-name
                - "{{.Cluster.ID}}"
+               - --network-plugin=cni
+               - --cni-conf-dir=/etc/cni/net.d
+               - --cni-bin-dir=/opt/cni/bin
```

Comment:
Are these flags defined at the right place?

Reply:
Given that we already run CNI for the EKS setup, we should be able to [peek here](https://github.com/zalando-incubator/kubernetes-on-aws/pull/7808/files#diff-fbbc8221d3bad7fc60e7c368f7da00a60c7e56a4060b29196a89e0834e42755c) for some inspiration on how to set it up.

Reply:
no, that is not the right place. this PR is wrong and should have been closed